README.md

```diff
@@ -32,43 +32,64 @@ for the zonemaster implementation are available.
 
 Also, in this repository, you can find a brief [user guide](USING.md).
 
-### Prerequisites
+## Prerequisites
 
 Zonemaster comes with documentation for and has been tested with these
```

Comment:
This sentence looks funny now, since the following sentence has been changed into a heading.